rollout-renku-version/rollout-renku-deployment.sh

```diff
@@ -45,8 +45,8 @@ do
     git checkout -b auto-update/${CHART_NAME}-${CHART_VERSION}-${cluster} ${UPSTREAM_BRANCH}
 
     yq w -i $cluster_dir/main/charts/renku.yaml "spec.chart.spec.version" $CHART_VERSION
-    sed -i "/Renku version/c\          ### Renku version $CHART_VERSION ($date)" $cluster_dir/main/charts/renku.yaml
-    sed -i "/Release Notes/c\          See the [Release Notes](https://github.com/${GITHUB_REPOSITORY}/releases/tag/$CHART_VERSION)" $cluster_dir/main/charts/renku.yaml
+    sed -i "/Renku version/c\            ### Renku version $CHART_VERSION ($date)" $cluster_dir/main/charts/renku.yaml
```

Comment:
Can we change the variable `date` to be `DATE` as the former is also a standard utility - when testing this, I thought incorrectly that it was using the command rather than referencing a variable - it would be clearer if we switched to upper case.

Reply:
I was using caps for input variables and lower caps for local variables, that's why I used `date`.
But I'm not too set on this so I'll just make the change you suggest.